Patch taken from the Desktop consolidation.
See BugDB CR #17795775
poppler-data,poppler,libspectre,evince- resolve build issue and add TPNO changes
for more details.

This patch will be submitted upstream if the problem still exists in the
latest version of poppler.

--- poppler-0.14.4/glib/demo/main.c.orig	2013-11-14 09:10:00.162047254 +0530
+++ poppler-0.14.4/glib/demo/main.c	2013-11-14 09:12:20.595665974 +0530
@@ -158,7 +158,7 @@ gint main (gint argc, gchar **argv)
 		return 1;
 	}
 
-	if (!g_thread_supported ())
+	if (!g_thread_get_initialized())
 		g_thread_init (NULL);
 
 	gtk_init (&argc, &argv);
